src/features/emojiMod.ts

```diff
@@ -44,6 +43,23 @@ type ReactionHandlers = {
   }) => void;
 };
 
+async function waitForThreadActivity(
```

Comment:
What's this change? Seems unrelated from the main intent of this PR, can this be pulled out into a separate change with its own description?